app/MMB/src/context/TimelineEventContext.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import * as React from 'react';
+import {
+  type TextPropsType,
+  Text,
+  StyleSheet,
+  Color,
+} from '@kiwicom/mobile-shared';
+
+const { Consumer, Provider } = React.createContext({
+  isPastEvent: false,
+});
+
+const TextWithContext = ({ children, style }: TextPropsType) => (
```

Comment:
Why is the consumer here?

Also It would be clearer to call it something like EventText (PastEventText) Something explaining what it is. TextWithContext is not really saying what this component is/does